src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Network Analysis/FindRoute/FindRoute.xaml

```diff
@@ -1,37 +1,34 @@
-﻿<UserControl 
+﻿<UserControl
     x:Class="ArcGISRuntime.UWP.Samples.FindRoute.FindRoute"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"    
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:esriUI="using:Esri.ArcGISRuntime.UI.Controls">
     <Grid>
-        <esriUI:MapView x:Name="MyMapView"/>
-        <Border Background="LightGray" Opacity=".85" BorderBrush="Black" BorderThickness="1"
-                HorizontalAlignment="Left" VerticalAlignment="Top"
+        <esriUI:MapView x:Name="MyMapView" />
+        <Border Background="WhiteSmoke" Opacity=".85" BorderBrush="Black" BorderThickness="1"
```

Comment:
Tip: You can use the translucent-brush trick from Animate3DGraphic here, to avoid having see-through text. You can also use an ARGB hex color in UWP: `Background="#D8F5F5F5"` (shorter but less readable approach).